medical_physician/models/medical_physician.py

```diff
@@ -38,6 +38,28 @@ class MedicalPhysician(models.Model):
         comodel_name='medical.physician.schedule.template',
         inverse_name='physician_id',
     )
+    medical_center_ids = fields.Many2many(
+        string='Primary Medical Center',
+        comodel_name='medical.center',
+        help="The physician's medical centers.",
+        compute='_compute_medical_center_ids',
+        store=True,
+        readonly=False,
+    )
+
+    @api.multi
+    @api.depends('parent_id')
+    def _compute_medical_center_ids(self):
+        for record in self:
+            center = self.env['medical.center'].search([
+                ('partner_id', '=', record.parent_id.id),
+            ], limit=1)
+
+            if not center:
+                continue
+
+            if center not in record.medical_center_ids:
+                record.medical_center_ids += center
```

Comment:
So I added a compute method for `medical_center_ids` with this section here. What's really weird is whenever I change `parent_id` in the UI, `medical_center_ids` is completely emptied and this single center record is added. Any ideas as to why that may be?